src/apps/reservation-list/modal/pause-reservation/pause-reservation.tsx

```diff
@@ -24,7 +24,7 @@ const PauseReservation: FC<PauseReservationProps> = ({ id, user }) => {
   const [startDate, setStartDate] = useState<string>(
     (config("pauseReservationStartDateConfig") as string) || ""
   );
-  const [endDate, setEndDate] = useState<string | null>("");
+  const [endDate, setEndDate] = useState<string>("");
```

Comment:
As I have mentioned in another comment I think would should have typed dates.
Why is the initial value not null? I didn't investigate further so you may very well have a good reason to have it like that.

Reply:
Because the date input does not work with null